libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1248,6 +1251,34 @@ void RC_Channel::do_aux_function_fft_notch_tune(const AuxSwitchPos ch_flag)
 #endif
 }
 
+/**
+ * Perform the RETRACT_MOUNT 1/2 process.
+ * 
+ * @param [in] ch_flag  Position of the switch. HIGH, MIDDLE and LOW.
+ * @param [in] instance 0: RETRACT MOUNT 1 <br>
+ *                      1: RETRACT MOUNT 2
+*/
+void RC_Channel::do_aux_function_retract_mount(const AuxSwitchPos ch_flag, const uint8_t instance)
+{
+#if HAL_MOUNT_ENABLED
+    AP_Mount *mount = AP::mount();
+    if (mount == nullptr || mount->get_num_instances() < (instance+1u)) {
```

Comment:
Mount `set_mode` call already deals with a invalid instance. There should be no need to check again here. That means we can save a little flash because we don't need the new `get_num_instances` method.

Reply:
I understand. I will delete it.